lib/ansible/modules/network/nxos/nxos_ospf_vrf.py

```diff
@@ -142,6 +150,9 @@
 from ansible.module_utils.netcfg import CustomNetworkConfig
 
 
+BOOL_PARAMS = [
```

Comment:
While I appreciate the effort, if you aren't going to use this var, there's no reason to add it.